assets/conf.toml

```diff
@@ -11,6 +12,8 @@
 #
 # Default:
 #   url = "http://localhost"
+url = "https://messenger.soc.stg.t11913.org"
+port = 443
```

Comment:
You shouldn't push any local configuration to the repository. This configuration is local only and is used for development. With that change being deployed the production server won't be used and therefore the end users may lose their accounts.